examples/counter_out/cont_gen_dig_pulse_train_buff_implicit.py

```diff
@@ -0,0 +1,32 @@
+"""Example for continuously generating digital pulse train with implicit timing.
+
+This example demonstrates how to generate a continuous buffered
+implicit timed digital pulse train from a Counter Output
+Channel. The Frequency, Duty Cycle, and Idle State are all
+configurable. The default data generated is a pulse train with a
+fixed frequency but a duty cycle that varies based on the Duty
+Cycle Max/Min and the signal type. The duty cycle will update
+with each sample generated.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, Level
+from nidaqmx.types import CtrFreq
+
+with nidaqmx.Task() as task:
+    duty_min = 0.5
+    duty_max = 0.8
+    duty_step = (duty_max - duty_min) / 1000
+    ctr_freq_data: CtrFreq = [CtrFreq(1000, (duty_min + duty_step * i)) for i in range(1000)]
```

Comment:
This type hint is wrong. The value is a `List[CtrFreq]` but the hint says it's a single `CtrFreq`. You don't need a type hint here at all because mypy can infer the type from the list comprehension.

Remove `: CtrFreq` from all of these examples.

Reply:
Okay, CtrFreq type hint is removed.